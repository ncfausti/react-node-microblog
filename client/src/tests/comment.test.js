/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Comment from '../components/comment';

test('Page matches snapshot', () => {
  const component = renderer.create(
    <Comment
      commentid={1}
      userid={31}
      avatar_ref={'https://wallpaperaccess.com/full/2213448.jpg'}
      username={'feng3116'}
      nickname={'Feng'}
      creation_date={'4/27/2021, 8:27:30 AM'}
      content={'test comment'}
      replying_to={'feng3116'}
      changeReplyingto={undefined}
      viewerid={31}
      delete={undefined}
    />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
