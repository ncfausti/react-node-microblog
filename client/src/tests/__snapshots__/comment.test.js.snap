// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page matches snapshot 1`] = `
<div
  className="comment"
>
  <div
    className="comment-avatar"
  >
    <img
      alt="avatar"
      src="https://wallpaperaccess.com/full/2213448.jpg"
    />
  </div>
  <div
    className="comment-main"
  >
    <div
      className="comment-main-nameplate"
    >
      <span
        className="comment-nickname"
      >
        Feng
      </span>
      <span
        className="comment-username"
      >
        @
        feng3116
      </span>
      <span
        className="comment-date"
      >
         Â· 
        4/27/2021, 8:27:30 AM
      </span>
      <div
        className="dropdown btn-group"
        id="post-dropdown"
        role="group"
      >
        <button
          aria-expanded={false}
          aria-haspopup={true}
          className="dropdown-toggle dropdown-toggle-split btn btn-secondary btn-sm"
          disabled={false}
          id="dropdown-split-basic"
          onClick={[Function]}
          type="button"
        />
      </div>
    </div>
    <div
      className="replying-to"
    >
      <span>
        Replying to @
        feng3116
      </span>
    </div>
    <div
      className="comment-main-content"
    >
      test comment
    </div>
  </div>
</div>
`;
